src/praisonai/praisonai/scheduler/deployment.py

```
@@ -175,6 +176,34 @@ def deploy_once(self) -> bool:
+            True if deployment succeeded, False otherwise
+        """
+        deployer = self._get_deployer()
+        
+        for attempt in range(max_retries):
+            try:
+                if deployer.deploy():
+                    logger.info(f"Deployment successful on attempt {attempt + 1}")
+                    return True
+                else:
+                    logger.warning(f"Deployment failed on attempt {attempt + 1}")
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)